doc/source/cookbook/amrkdtree_downsampling.py

```diff
@@ -71,8 +71,13 @@
     alpha=10.0 * np.ones(4, dtype="float64"),
     colormap="RdBu_r",
 )
-tf.add_layers(4, 0.01, col_bounds=[-27.5, -25.5],
-              alpha=10.0 * np.ones(4, dtype='float64'), colormap='RdBu_r')
+tf.add_layers(
```

Comment:
this showed up in another PR too as an effect of the slash command "format", but it doesn't correspond to a problem seen by black. I checked and this is not even a file we ask black to ignore so I don't really get why it does this. @matthewturk

Reply:
yeah, I didn't understand why that showed up

Reply:
Maybe it's running with a different config.  The invocation is just `black` I think.

https://github.com/yt-project/slash-command-processor/blob/master/.github/workflows/black-command.yml
https://github.com/yt-project/slash-command-processor/blob/master/.github/workflows/format-command.yml

Reply:
also, I manually ran `black` as well, since it wasn't clear to me that the command worked the first time.